public/client.js

```diff
@@ -38,33 +38,31 @@ function gameSetup() {
   };
 }
 
+/**
+ * Chrome Android browser does not like it when
+ * 
+ * background image positions are set to 100%. capping this to 99.6%, which seems ok
+ * 
+ * @param {number} idx
+ * 
+ * @return {number}
+ */
+function backgroundPosition(idx) {
```

Comment:
```suggestion
function backgroundPosition(idx) {
```

Saw the same logic twice. Threw it into a function.

Reply:
ahhh I'm not so wild about this either; I sometimes think it's clearer to just be explicit and doing this moves the logic away from the place it's used. It's only twice :D